src/js/structs/Job.js

```diff
@@ -9,4 +22,7 @@ module.exports = class Job extends Item {
     return this.getId().split('/').pop();
   }
 
+  getSchedule() {
+    return this.get('schedule');
```

Comment:
It's going to be called `schedules` and it's an array. Please refer to the design doc as it's been recently updated.